python/astshim/fitsChan.cc

```diff
@@ -24,6 +24,7 @@
 
 #include <pybind11/pybind11.h>
 #include <pybind11/complex.h>
+#include <pybind11/stl.h>
```

Comment:
It is needed because `getCardNames` returns a vector. `pybind11/stl.h` wraps C++ vectors so they are usable in Python.